docs/UserGuide.md

```diff
@@ -664,64 +785,73 @@ VendorVault automatically saves your data as a JSON file `[JAR file location]/da
-**[List](#listing-all-suppliers-list-s)**   | `list -s`
-**[Mark](#mark-a-supplier-with-a-status-mark-s)**   | `mark -s INDEX STATUS`<br> e.g.,`mark -s 2 active`
-**[Find](#find-a-supplier-find-s)**   | `find -s n/KEYWORD_FOR_SUPPLIER NAME> com/KEYWORD_FOR_SUPPLIER_COMPANY pro/<KEYWORD_FOR_SUPPLIER_PRODUCT>` <br> e.g., `find -s n/link com/NU`
-**[Sort](#sort-suppliers-sort-s)**   | `sort -s so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -s so/a sb/n`
+**[Add]()**    | `add -s n/NAME p/PHONE e/EMAIL com/COMPANY [t/TAG]…​ [pro/PRODUCT]…​` <br> e.g., `add -s n/John Doe p/98765432 e/johnd@example.com com/companyA t/friends t/owesMoney pro/rice pro/bread`
+**[Delete]()** | `delete -s INDEX`<br> e.g., `delete -s 3`
+**[List]()**   | `list -s`
+**[Mark]()**   | `mark -s INDEX STATUS`<br> e.g.,`mark -s 2 active`
+**[Find]()**   | `find -s n/KEYWORD_FOR_SUPPLIER NAME> com/KEYWORD_FOR_SUPPLIER_COMPANY pro/<KEYWORD_FOR_SUPPLIER_PRODUCT>` <br> e.g., `find -s n/link com/NU`
+**[Sort]()**   | `sort -s so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -s so/a sb/n`
 
 
-### Delivery-specific Commands
+## 7.3 Delivery-specific Commands
 
 Action     | Format, Examples
 -----------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------
-**[Add](#adding-a-delivery-add-d)**    | `add -d on/DELIVERY_DATE_TIME s/SUPPLIER_INDEX pro/PRODUCT q/QUANTITY kg/g/L/mL/units c/COST` <br> e.g., `add -d on/18-06-2024 17:00 s/1 pro/bread q/500 g c/5.50`
-**[Delete](#deleting-a-delivery-delete-d)** | `delete -d INDEX`<br> e.g., `delete -d 3`
-**[List](#listing-all-deliveries-list-d)**   | `list -d`
-**[Mark](#marking-a-delivery-mark-d)**   | `mark -d INDEX STATUS`<br> e.g.,`mark -d 2 PENDING`
-**[Find](#find-a-delivery-find-d)**   | `find -d on/DELIVERY_DATE_TIME stat/STATUS s/SUPPLIER_INDEX pro/PRODUCT`<br> e.g., `find -d on/ 28-06-2025 17:00 pro/ milk`
-**[Sort](#sort-deliveries-sort-d)**   | `sort -d so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -d so/a sb/c`
-**[Upcoming](#upcoming-deliveries-upcoming)** | `upcoming aft/START_DATE bef/END_DATE`<br> e.g., `upcoming aft/19-12-2022 08:00 bef/18-06-2023 17:00`
+**[Add]()**    | `add -d on/DELIVERY_DATE_TIME s/SUPPLIER_INDEX pro/PRODUCT q/QUANTITY kg/g/L/mL/units c/COST` <br> e.g., `add -d on/18-06-2024 17:00 s/1 pro/bread q/500 g c/5.50`
+**[Delete]()** | `delete -d INDEX`<br> e.g., `delete -d 3`
+**[List]()**   | `list -d`
+**[Mark]()**   | `mark -d INDEX STATUS`<br> e.g.,`mark -d 2 PENDING`
+**[Find]()**   | `find -d on/DELIVERY_DATE_TIME stat/STATUS s/SUPPLIER_INDEX pro/PRODUCT`<br> e.g., `find -d on/ 28-06-2025 17:00 pro/ milk`
+**[Sort]()**   | `sort -d so/SORT_ORDER sb/SORT_BY_FIELD`<br> e.g., `sort -d so/a sb/c`
+**[Upcoming]()** | `upcoming aft/START_DATE bef/END_DATE`<br> e.g., `upcoming aft/19-12-2022 08:00 bef/18-06-2023 17:00`
```

Comment:
Need change this as well to DELIVERY_DATE_TIME instead of start date and end_Date